homeassistant/components/permobil/coordinator.py

```diff
@@ -0,0 +1,57 @@
+"""DataUpdateCoordinator for permobil integration."""
+
+from datetime import timedelta
+import logging
+
+import async_timeout
+from mypermobil import (
+    ENDPOINT_BATTERY_INFO,
+    ENDPOINT_DAILY_USAGE,
+    ENDPOINT_VA_USAGE_RECORDS,
+    MyPermobil,
+    MyPermobilAPIException,
+    MyPermobilClientException,
+)
+
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryAuthFailed
+from homeassistant.helpers.update_coordinator import (
+    DataUpdateCoordinator,
+    UpdateFailed,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class MyPermobilCoordinator(DataUpdateCoordinator):
```

Comment:
Set the expected return type here as well:

```suggestion
class MyPermobilCoordinator(DataUpdateCoordinator[<return type>]):
```